### ISSUE TITLE
doc suggestion re circumventing prototypes

### ISSUE THREAD

**Issue by y** [garu/Data-Printer#73]
In the sections about circumventing prototypes, you might want to also include the trick of dereffing an anonymous structures to make it work with p. For example:

```
p %{ { foo => 123 } };
p @{ [ 1, 2, 3] };
```

Also, when 5.24 is released, postderef will no longer be experimental, and this is a little nicer:

```
p { foo => 123 }->%*;
p [ 1, 2, 3]->@*;
```

**Comment by jayallen** [comment]
Wow, I didn't know that was possible and I love it so much.  Thank you @y!

**Comment by garu** [comment]
Hi @y! Thank you so much for the contribution! Took my a while (sorry!) but it's on the master branch, soon to be released as Data::Printer 1.0.

Thanks again!
